app/assets/stylesheets/actiontext.scss

```diff
@@ -9,6 +9,10 @@
 // <action-text-attachment> element we wrap around attachments. Otherwise,
 // images in galleries will be squished by the max-width: 33%; rule.
 .trix-content {
+  a {
+    color: blue;
+  }
+
```

Comment:
I had to make this change because links in the rich text field weren't being highlighted in a typical web link blue color. Everything with embedded web links worked fine except for the coloring.

If there is a better way of fixing this, please let me know and I'll try a different way.

Reply:
I have limited experience with the Trix editor, so I'am not sure.

Reply:
I think it should be fine.